cookbooks/aws-parallelcluster-platform/templates/enroot/enroot.conf.erb

```diff
@@ -1,9 +1,9 @@
 #ENROOT_LIBRARY_PATH        /usr/lib/enroot
 #ENROOT_SYSCONF_PATH        /etc/enroot
-ENROOT_RUNTIME_PATH        /tmp/enroot/user-$(id -u)
-ENROOT_CONFIG_PATH         ${ENROOT_CONFIG_PATH}
-ENROOT_CACHE_PATH          ${ENROOT_CACHE_PATH}
-ENROOT_DATA_PATH           /tmp/enroot/data/user-$(id -u)
+ENROOT_RUNTIME_PATH        /run/enroot/user-$(id -u)
+ENROOT_CONFIG_PATH         
```

Comment:
Maybe we should create an enroot config path. Like `/home/user-$(id -u)/.config/enroot`

Reply:
Our users could share the /home folder and we would end up in the original situation where these paths expected to be local became shared. I suggest to avoid the use of /home for these paths. In Pyxis/Enroot docs/Example sometimes they mention /home because they assume it's a local folder.